docs/content/deployment/dagster-daemon.mdx

```diff
@@ -1,36 +1,183 @@
 
 The following daemons are currently available:
 
-- The _scheduler daemon_ is responsible for creating runs from any [schedules](/concepts/partitions-schedules-sensors/schedules) that are turned on. This daemon will run as long as you have not overridden the default <PyObject module="dagster._core.scheduler" object="DagsterDaemonScheduler" /> as the scheduler on your instance.
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
+    <tr>
+      <th
+        style={{
+          width: "20%",
+        }}
+      >
+        Name
+      </th>
+      <th>Description</th>
+      <th>Enabled by</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Scheduler daemon</td>
+      <td>
+        Creates runs from active{" "}
+        <a href="/concepts/partitions-schedules-sensors/schedules">schedules</a>
+      </td>
+      <td>
+        Enabled / runs as long as the default{" "}
+        <PyObject
+          module="dagster._core.scheduler"
+          object="DagsterDaemonScheduler"
+        />{" "}
+        isn't overriden as the scheduler on your instance.
+      </td>
+    </tr>
+    <tr>
+      <td>Run queue daemon</td>
+      <td>
+        Launches queued runs, taking into account any limits and prioritization
+        rules set on your instance
+      </td>
+      <td>
+        Setting the <a href="/deployment/run-coordinator">run coordinator</a> on
+        your instance to{" "}
+        <PyObject
+          module="dagster._core.run_coordinator"
+          object="QueuedRunCoordinator"
+        />
+        .
+      </td>
+    </tr>
+    <tr>
+      <td>Sensor daemon</td>
+      <td>
+        Creates runs from active{" "}
+        <a href="/concepts/partitions-schedules-sensors/sensors">sensors</a>{" "}
+        that are turned on
+      </td>
+      <td>Always enabled</td>
+    </tr>
+    <tr>
+      <td>Run monitoring daemon</td>
+      <td>
+        Handles <a href="/deployment/overview#job-execution-flow">run worker</a>{" "}
+        failures
+      </td>
+      <td>
+        Using the <code>run_monitoring</code> field in your instance. Refer to
+        the{" "}
+        <a href="/deployment/run-monitoring">Run monitoring documentation</a>{" "}
+        for more info.
+      </td>
+    </tr>
+  </tbody>
+</table>
 
-- The _run queue daemon_ is responsible for launching queued runs, taking into account any limits and prioritization rules you've set on your instance. You can enable this daemon by setting the [run coordinator](/deployment/run-coordinator) on your instance to <PyObject module="dagster._core.run_coordinator" object="QueuedRunCoordinator" />.
+Daemons that need to load code location(s) will do so by periodically reloading your instance's [workspace file](/concepts/code-locations/workspace-files). This means that the `dagster-daemon` process doesn't need to be restarted when workspace files are changed.
```

Comment:
This makes it sound like the only way to load code locations is through `workspace.yaml`. It should be prefaced with, "if you have configured your daemon to use the workspace.yaml file" or something.

Also would like to hear @gibsondan's thoughts on this as I might be missing something.